### ISSUE TITLE
Use different width and height

### ISSUE THREAD

**Issue by zidein07** [alexk111/ngImgCrop#136]
Hello guys, i want using different width and height for my image crop is it possible?

**Comment by ignacio-chiazzo** [comment]
Take a look at this example https://github.com/ignacio-chiazzo/ngImgCropTool-Examples.

**Comment by arturskeeled** [comment]
@ignacio-chiazzo hey! i was trying to replicate your code, but for some reason i can't.
what i did was copy it to the original live demo in [js fiddle](http://jsfiddle.net/alexk111/rw6q9/) and from what i gather "rectangle" is not a valid name, so it defaults back to "circle". Help would be much appreciated. Regards.

**Comment by 88media** [comment]
@ignacio-chiazzo & @arturskeeled , I am trying to do exactly the same. Would be useful! I also see aspect-ratio="1.7", but this doesn't seem to do anything. Thanks!

**Comment by albertonaperijr** [comment]
Try to use the extended version since it is in active development: https://github.com/CrackerakiUA/ngImgCropFullExtended/

**Comment by 88media** [comment]
@albertonaperijr, thanks for that version!

**Comment by ignacio-chiazzo** [comment]
@88media @arturskeeled You are right!, the version that I was using is not up to date, before this directive I was using `rectangle` instead of `square`. To use a rectangle just add `_area-type="square"` , take a look at [fiddle](https://jsfiddle.net/ignaciochiazzo/t28uL57n/). 

Feel free to make a PR
